SQL/0000-00-00-schema.sql

```diff
@@ -637,7 +636,6 @@ CREATE TABLE `files` (
   KEY `sessionid` (`SessionID`),
   KEY `outputtype` (`OutputType`),
   KEY `filetype_outputtype` (`FileType`,`OutputType`),
-  KEY `staging_filetype_outputtype` (`PendingStaging`,`FileType`,`OutputType`),
```

Comment:
should there also be a cleanup patch for existing LORIS instances?

Reply:
@driusan There definitely should be and I created it on Friday. I simply forgot to commit it to the branch... Good catch! Thanks!! Should be fixed now.